src/screens/Profiles/UserLibrary/components/UserLibraryScreen/component.js

```diff
@@ -320,4 +320,4 @@ export class UserLibraryScreenComponent extends React.Component {
       </View>
     );
   }
-}
+}
```

Comment:
if you install an editor config plugin for your editor, new liens will be magically added for you :D